www/docs/mp/index.php

```diff
@@ -492,27 +494,22 @@ function ($k) { return $k['score_difference'] >= 2; }
             $memcache->set($mem_key, $highlighted_for_this_mp, 60 * 60 * 24);
         }
 
+        $data['og_image'] = \MySociety\TheyWorkForYou\Url::generateSocialImageUrl($member_name, 'Election Register', $data['current_assembly']);
         $data['mp_has_highlighted_interests'] = (bool) $highlighted_for_this_mp;
         $overlapping_interests = [];
 
         MySociety\TheyWorkForYou\Renderer::output('mp/election_register', $data);
 
         // no break
     case 'register':
+        $data['og_image'] = \MySociety\TheyWorkForYou\Url::generateSocialImageUrl($member_name, 'Register of Interests', $data['current_assembly']);
         // Send the output for rendering
         MySociety\TheyWorkForYou\Renderer::output('mp/register', $data);
 
         // no break
-    case 'policy_set_svg':
```

Comment:
given we're removing this we should remove the code under votes that sets the og_image to use these as it now links to the profile page.